### PR TITLE
Avoid to execute natively when ignoring expressions

### PR DESCRIPTION
**Pull request by darroyocazorla**

### CODE REVIEW

**Comment by miguel0afd** [comment]
:+1:

File: core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ExtendedDataSourceStrategy.scala [hunk 1]
```diff
@@ -62,8 +62,9 @@ private[sql] object ExtendedDataSourceStrategy extends Strategy with Logging {
 
     val (pro, fil, att2udf) =
       (CatalystToCrossdataAdapter.getConnectorLogicalPlan(plan, projects, filterPredicates): @unchecked) match {
-        case (_, FilterReport(_, udfsIgnored)) if udfsIgnored.nonEmpty => cannotExecuteNativeUDF(udfsIgnored)
-        case (SimpleLogicalPlan(pro, fil, udfs, _), _) => (pro, fil, udfs)
+        case (_, _, FilterReport(_, udfsIgnored)) if udfsIgnored.nonEmpty =>
+          cannotExecuteNativeUDF(udfsIgnored)
+        case (SimpleLogicalPlan(pro, fil, udfs, _), _, _) => (pro, fil, udfs)
```
Comment (compae) [inline core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ExtendedDataSourceStrategy.scala]: can you change the description of vars? pro, fil?

Comment (darroyocazorla) [inline core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ExtendedDataSourceStrategy.scala]: pfcoperezRef forward compaeMessage

Comment (pfcoperez) [inline core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ExtendedDataSourceStrategy.scala]: @compae I'd merge the PR and change the identifiers later.

Comment (miguel0afd) [inline core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ExtendedDataSourceStrategy.scala]: Agreed with @pfcoperez!

**Pull request merged**